java/ql/lib/ext/io.netty.handler.codec.http2.model.yml

```diff
@@ -0,0 +1,33 @@
+extensions:
+- addsTo:
+    pack: codeql/java-all
+    extensible: sourceModel
+  data:
+    - ["io.netty.handler.codec.http2", "Http2FrameListener", True, "onDataRead", "", "", "Parameter[2]", "remote", "manual"]
+    - ["io.netty.handler.codec.http2", "Http2FrameListener", True, "onHeadersRead", "", "", "Parameter[2]", "remote", "manual"]
+    - ["io.netty.handler.codec.http2", "Http2FrameListener", True, "onPushPromiseRead", "", "", "Parameter[3]", "remote", "manual"]
```

Comment:
Should we consider the `Http2Settings` parameter of [`Http2FrameListener.onSettingsRead`](https://netty.io/4.1/api/io/netty/handler/codec/http2/Http2FrameListener.html#onSettingsRead-io.netty.channel.ChannelHandlerContext-io.netty.handler.codec.http2.Http2Settings-) a remote source as well?

Reply:
Good point, I thought the keys of the map were strings. In that case it's correct to not add it as source.